admin/src/main/resources/io/buoyant/admin/js/help-page.js

```diff
@@ -0,0 +1,15 @@
+$(function() {
+  var helpText = '<h1 class="title">Need some help?</h1>' +
```

Comment:
The help page on linkerd.io was updated recently, I think this is missing those changes.